searx/engines/brave.py

```diff
@@ -152,6 +152,10 @@
 paging = False
 """Brave only supports paging in :py:obj:`brave_category` ``search`` (UI
 category All)."""
+max_page = 6
```

Comment:
I can request up to 9 pages on brave: https://search.brave.com/search?q=the&offset=8&spellcheck=0

Could you set max page to 10 instead?

related: https://github.com/searxng/searxng/issues/2982#issuecomment-1810679732